pkg/bulk/cmd_test.go

```diff
@@ -21,117 +22,67 @@ type bulkTester struct {
 	err     error
 	opErr   error
 
-	infos    []runtime.Object
 	recorded []runtime.Object
 }
 
 func (bt *bulkTester) ResourceSingularizer(resource string) (string, error) {
 	return resource, nil
 }
 
-func (bt *bulkTester) InfoForObject(obj runtime.Object, preferredGVKs []schema.GroupVersionKind) (*resource.Info, error) {
-	bt.infos = append(bt.infos, obj)
-	// These checks are here to make sure the preferredGVKs are set to retain the legacy
-	// behavior for bulk operations.
-	if len(preferredGVKs) == 0 {
-		return nil, fmt.Errorf("expected preferred gvk to not be empty")
-	}
-	if preferredGVKs[0].Group != "" {
-		return nil, fmt.Errorf("expected preferred gvk to be set to prefer the legacy group")
-	}
-	return &resource.Info{Object: obj, Mapping: bt.mapping}, bt.err
-}
-
-func (bt *bulkTester) Record(info *resource.Info, namespace string, obj runtime.Object) (runtime.Object, error) {
+func (bt *bulkTester) Record(obj *unstructured.Unstructured, namespace string) (*unstructured.Unstructured, error) {
 	bt.recorded = append(bt.recorded, obj)
 	return obj, bt.opErr
 }
 
 func TestBulk(t *testing.T) {
 	bt := &bulkTester{
-		mapping: &meta.RESTMapping{
-			MetadataAccessor: meta.NewAccessor(),
-		},
+		mapping: &meta.RESTMapping{},
 	}
-	b := Bulk{Mapper: bt, Op: bt.Record}
+	b := Bulk{Scheme: legacyscheme.Scheme, Op: bt.Record}
```

Comment:
@deads2k can you layout some ground rules when we should use the `legacyscheme.Scheme` vs the one the other present in kubectl/oc? //cc @juanvallejo

Reply:
And as a continuation when to use `ocscheme.ReadingInternalScheme` vs `ocscheme.PrintingInternalScheme`.

Reply:
> @deads2k can you layout some ground rules when we should use the legacyscheme.Scheme vs the one the other present in kubectl/oc? //cc @juanvallejo

For the rebase crash-landing, do whatever works.  Long term, everything should use only external or unstructured schemes.